tools/cmake/mbed_greentea.cmake

```diff
@@ -70,6 +81,17 @@ macro(mbed_greentea_add_test)
 
     mbed_set_post_build(${MBED_GREENTEA_TEST_NAME})
 
+    add_test(
+        NAME ${MBED_GREENTEA_TEST_NAME}
+        COMMAND mbedhtrun
+            -m ${MBED_TARGET}
+            -f ${CMAKE_CURRENT_BINARY_DIR}/${MBED_GREENTEA_TEST_NAME}.bin
+            -p ${MBED_TARGET_SERIAL_PORT}
```

Comment:
This will default to 9600 baud but some targets have 115200 (e.g. ARM_MUSCA_S1). We can get it from MBED_CONF_PLATFORM_STDIO_BAUD_RATE.

htrun supports two ways to specify baud rate:
* `-p <port> --baud-rate=<baud>`
* `-p <port>:<baud>`

Reply:
I've added another commit to fetch the baud rate from the config. It's not pretty, because we don't define a variable for the `MBED_CONF ...` macros. They're just string values appended to a CMake list, so we have to scrape the baud rate from the list. Please review.